webclient/app/src/app/features/apps/entities/DefaultEntities.js

```diff
@@ -94,6 +94,10 @@ function toDatabaseEntity(entity) {
             if (field.fieldValidateRulesPattern) {
                 field.fieldValidateRules.push("pattern");
             }
+
+            if (field.fieldType !== "Enum") {
+                field.enumDef = null;
```

Comment:
Wird dieses Value in einem Select benutzt? Dann bitte auf "" setzen. Ansonsten kommt ne warning

Reply:
Ist das geklärt?

Reply:
Der value wird nicht benutzt, wenn der FieldType nicht Enum ist. Das Select wird nicht generiert.

Reply:
Okey, dann passt das!